### PR TITLE
Create to do list page component - Vanilla.js 

### PR DESCRIPTION
**Pull request by 29deepanshutyagi**
issue #87

### CODE REVIEW

File: javascript/dwa-starter-vanillajs-vite/components.js [hunk 1]
```diff
@@ -1,3 +1,136 @@
...
+  async createTask(task) {
+      // Placeholder for creating a task in DWN
+  },
+  async updateTask(task) {
+      // Placeholder for updating a task in DWN
+  },
+  async deleteTask(recordId) {
+      // Placeholder for deleting a task in DWN
+  }
+};
```
Comment (blackgirlbytes) [inline javascript/dwa-starter-vanillajs-vite/components.js:117]: Hi @29deepanshutyagi ! Thanks for your contribution, but this PR seems to be missing a few things. For example, these methods have no logic in them..just a comment. We'll need to implement the logic for DWAs. The best place to look and understand how these work are in the DWA react vite project and in the DWA vue project.

Also the task and criteria say the following..I don't think I see that happening here:
Create a TaskItem.vanilla component for individual task display and actions

Create a TaskForm.vanilla component for adding/editing tasks

Implement TodoDwnRepository class for DWN operations:

listTasks()

createTask(task)

updateTask(task)

deleteTask(recordId)

findTaskRecord(recordId)

listTasksRecords()

File: javascript/shared-todo-starter/package.json [hunk 2]
```diff
@@ -12,14 +12,23 @@
...
     "@playwright/test": "^1.44.1",
     "@tailwindcss/forms": "0.5.9",
     "@tailwindcss/typography": "0.5.15",
     "@types/node": "20.16.12",
     "nuxt": "3.13.2",
+<<<<<<< HEAD
+    "sass": "1.79.5",
+=======
     "sass": "1.80.4",
+>>>>>>> ea24aa69758146611ddc3c604b61c014494d51f9
```
Comment (blackgirlbytes) [inline javascript/shared-todo-starter/package.json:31]: hmm i dont think you want those lines in the package.json. it looks like you have merge conflicts and duplicates

**Review by blackgirlbytes — CHANGES_REQUESTED** [review]

File: javascript/shared-todo-starter/package.json (hunk 2 above)
Comment (blackgirlbytes) [reply javascript/shared-todo-starter/package.json:31]: Actually this file should have 0 changes as it's unrelated to your project

File: javascript/shared-todo/package.json [hunk 3]
```diff
@@ -11,14 +11,23 @@
...
     "@tailwindcss/forms": "0.5.9",
     "@tailwindcss/typography": "0.5.15",
     "@types/node": "20.16.12",
     "nuxt": "3.13.2",
+<<<<<<< HEAD
+    "sass": "1.79.5",
+=======
     "sass": "1.80.4",
+>>>>>>> ea24aa69758146611ddc3c604b61c014494d51f9
     "sass-loader": "16.0.2",
```
Comment (blackgirlbytes) [inline javascript/shared-todo/package.json:31]: this file should have 0 changes as it's unrelated to your project

**Comment by blackgirlbytes** [comment]
oh no looks like this PR has a few changes in projects that aren't related to your own. Let's get that fixed up. If you need help, come to our office hours in Discord!

**Comment by taniashiba** [comment]
@29deepanshutyagi - Hey there! Since Hacktoberfest officially closes tomorrow, we want to make sure all open PRs are resolved in full today in case of any issues. Please see comments above so you can get credit!

**Comment by blackgirlbytes** [comment]
hey @29deepanshutyagi im going to close this one out because we only have a few hours left until hacktoberfest ends, and I haven't heard from you!

**Pull request closed without merging**